### DISCUSSION TITLE
Control RGB LED for Sunton ESP32-3248S035C

### DISCUSSION THREAD

**Discussion by wilwizard1979** [HASwitchPlate/openHASP#461] [Q&A] [answered]
Hi Everyone,

Is there any way to control the RGB Led for Sunton ESP32-3248S035C?

This is what I found in the schematics:
![image](https://user-images.githubusercontent.com/108979356/230632489-8496ca08-423c-4865-a093-5e712c8d6a79.png)

Thanks in advance!

**Comment by fvanroie** [comment, marked as answer]
Configure the output pins in the [GPIO Settings](https://www.openhasp.com/0.7.0/firmware/configuration/gpio/#moodlight) as `Mood Red`, `Mood Green` and `Mood Blue`.
Then you can control the LEDs with the [`moodlight`](https://www.openhasp.com/0.7.0/commands/#moodlight) command.

**Comment by wilwizard1979** [comment]
Thanks @fvanroie,

It works! This is the configuration I had to make in my Display to make it work:

![image](https://user-images.githubusercontent.com/108979356/230807308-a2c7603f-5f0d-4108-8a08-77d68713d09e.png)

**Comment by alfem** [comment]
Where are the "mood" type GPIO pins?  I can not find them after an update.

![imagen](https://github.com/HASwitchPlate/openHASP/assets/697599/12a66788-f362-43dd-ba54-26ad795f4e30)

**Reply by fvanroie** [reply]
Check the output pins instead of the input pins.

**Reply by alfem** [reply]
Sorry! I figured that out 10 seconds after sending my post 

![imagen](https://github.com/HASwitchPlate/openHASP/assets/697599/f8be141b-b433-44be-8043-80567af54af1)
